main/run.py

```diff
@@ -26,19 +27,93 @@
     URBAN_BRIGHTNESS_THRESHOLD,
     URBAN_NEARBY_RADIUS,
     MIN_NEARBY_RURAL_STATIONS,
-    START_DATE,
-    END_DATE,
-    BASELINE_START_DATE,
-    BASELINE_END_DATE,
     SST_CUTOFF_TEMP,
+    URBAN_ADJUSTMENT_OPTION,
 )
 
 
+def parse_arguments():
+    """
+    Parse command-line arguments for executing the GISTEMP algorithm.
+
+    Returns:
+        argparse.Namespace: Parsed command-line arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Execute steps of the GISTEMP algorithm."
+    )
+    parser.add_argument(
+        "--urban_adjustment_option",
+        type=bool,
+        default=URBAN_ADJUSTMENT_OPTION,
+        help="Option to include urban adjustment (baseline does not include urban adjustment)",
+    )
+    parser.add_argument(
+        "--start_year",
+        type=int,
+        default=START_YEAR,
+        help="Start year for processing data",
+    )
+    parser.add_argument(
+        "--end_year", type=int, default=END_YEAR, help="End year for processing data"
```

Comment:
nit picky but the list format here is different than the rest of arguments (new line after each comma)

Reply:
This is due to the code formatter I'm using (called Black), which auto formats files into PEP8 format whenever you save. If the line was slightly longer, it would format into new lines after each comma.